web-client/src/components/MainLayout/index.tsx

```diff
@@ -70,7 +68,7 @@ const MainLayout = observer(({ children }: IMainLayout) => {
                     <AppBar component="nav" style={{ backgroundColor: 'var(--white-color)' }}>
                         <Toolbar style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
                             <NavLinkLogo to="/"><img src={Logo} alt="Logo" style={{ marginTop: '12px' }} /></NavLinkLogo>
-                            <FlexBox justify="space-between" width="360px">
+                            <FlexBox justify="space-between" width={language==='en' ? "360px" : "600px"}>
```

Comment:
why you have different width based on the language en?
Will these numbers like 100px vs 140 and 360px vs 600px be reused somewhere?

Reply:
1.In Ukrainian, these words are longer, and this block needs a different width for better positioning
2.No, only in these places

Reply:
Ah alright 👍